src/components/generic/movies/MoviesPanel.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react'
+import {FC} from 'react'
+import * as Movies from './style'
+import {MovieCard} from './MovieCard'
+import ReactPlaceholder from 'react-placeholder'
+import 'react-placeholder/lib/reactPlaceholder.css'
+import {InfiniteData} from 'react-query/types/core/types'
+
+interface Props {
+  moviePages: InfiniteData<any> | undefined
+  isLoading: boolean
+}
+
+const placeholderStyle = {
+  width: '200px',
+  height: '346px',
+  marginTop: '10px',
+  borderRadius: '5px',
+}
+
+export const MoviesPanel: FC<Props> = ({moviePages, isLoading}) => {
+  return (
+    <Movies.Movies>
+      {isLoading ? RenderLoading(isLoading) : RenderMovies(moviePages)}
+    </Movies.Movies>
+  )
+}
+
+const RenderMovies = (moviePages: InfiniteData<any> | undefined) => {
+  //Fix feature:
```

Comment:
```suggestion
  // TODO: fix feature
```

`TODO`s are easier to find and remember.